doc/widgets/labeling.md

```diff
@@ -0,0 +1,21 @@
+# Peak Labeling
+
+The peak labeling menu enables allows for individual or automatic peak labeling.
```

Comment:
Reduce wordiness and try to make writing clearer. Perhaps use Grammarly for help?